main/src/ca/uwaterloo/flix/language/phase/Stratifier.scala

```diff
@@ -78,7 +78,7 @@ object Stratifier extends Phase[Root, Root] {
     */
   private def visitDef(def0: Def)(implicit dg: DependencyGraph, cache: Cache): Validation[Def, CompilationMessage] =
     visitExp(def0.impl.exp) map {
-      case e => def0.copy(impl = def0.impl.copy(exp = e))
```

Comment:
We always use case. I know IDEA suggests the opposite, but using case makes it easier when refactoring the code.

Feel free to add a line to `STYLE.md`.

Reply:
Reason: If you change from a single variable to anything else you *must* use case.

Reply:
I will case it (and my settings)